### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by p0intbl4nc**
poli sanam

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # fossworkshop
 hello
 how are you
+heyyybroooiii
```
Comment (jrvarma8) [inline README.md:4]: nice man

Comment (vinayak-byte) [reply README.md:4]: ikr

Comment (vinayak-byte) [reply README.md:4]: sup bois

Comment (sagarsaji) [reply README.md:4]: bigileeee

**Pull request merged**